app/controllers/turbo/frames/frame_request.rb

```diff
@@ -1,19 +1,24 @@
 # Turbo frame requests are requests made from within a turbo frame with the intention of replacing the content of just
 # that frame, not the whole page. They are automatically tagged as such by the Turbo Frame JavaScript, which adds a
-# <tt>Turbo-Frame</tt> header to the request. When that header is detected by the controller, we ensure that any
-# template layout is skipped (since we're only working on an in-page frame, thus can skip the weight of the layout), and
-# that the etag for the page is changed (such that a cache for a layout-less request isn't served on a normal request
-# and vice versa).
+# <tt>Turbo-Frame</tt> header to the request.
 #
-# This is merely a rendering optimization. Everything would still work just fine if we rendered everything including the layout.
-# Turbo Frames knows how to fish out the relevant frame regardless.
+# When that header is detected by the controller, we substitute our own minimal layout in place of the
```

Comment:
What about mentioning the minimal layout is `turbo_rails/frame.html.erb`? People can still override the default if the want creating their own `app/views/layouts/turbo_rails/frame.html.erb` file. Probably not a very common case, but still doable if someone wants to do it.

Reply:
> What about mentioning the minimal layout is `turbo_rails/frame.html.erb`? People can still override the default if the want creating their own `app/views/layouts/turbo_rails/frame.html.erb` file.

I like that idea. Action Text's documentation makes a similar suggestion for folks needing to customize the default layout in https://guides.rubyonrails.org/action_text_overview.html#rendering-rich-text-content